pkg/cli/webhook/github.go

```
@@ -49,19 +48,18 @@ func (gh *gitHubConfig) Run(ctx context.Context, opts *Options) (*response, erro
 }
 
 func (gh *gitHubConfig) askGHWebhookConfig(repoURL, controllerURL, apiURL string) error {
-	var defaultRepo string
-	if repoURL != "" {
-		defaultRepo, _ = formatting.GetRepoOwnerFromURL(repoURL)
-	}
-
-	if repoURL == "" || defaultRepo == "" {
-		msg := "Please enter the repository you want to be configured (eg. repo-owner/repo-name) : "
-		if err := prompt.SurveyAskOne(&survey.Input{Message: msg}, &defaultRepo,
+	if repoURL == "" {
+		msg := "Please enter the git repository url you want to be configured: "
+		if err := prompt.SurveyAskOne(&survey.Input{Message: msg}, &repoURL,
 			survey.WithValidator(survey.Required)); err != nil {
 			return err
 		}
+	} else {
+		fmt.Fprintf(gh.IOStream.Out, "✓ Setting up GitHub Webhook for Repository %s\n", repoURL)
 	}
 
+	defaultRepo, _ := formatting.GetRepoOwnerFromURL(repoURL)
```

what about not ignoring the error, to be able to validate it's a good repoURL ?